rfcs/0000-sudt-info/0000-sudt info.md

```diff
@@ -0,0 +1,184 @@
+
+The type script of the info cell checks whether the transaction satisfies one of the two logics(single-owner or script-driven) above.
+
+## 4. info data in witness
+
+The above describes a solution to put info data in the cell, but if we don’t change info often, we can treat sudt Info as static data and put it in witness. This solution is an additional solution to info_cell and does not conflict with info_cell.
+
+The advantage is that it can reduce the state occupation on the chain. When sudt is useless, no state will be occupied, and there will be no situation where the state is still occupied on the chain after the EOS fundraising ends. The disadvantage is that if the blockchain of ckb is to be prune in the future, the information may be lost.
+
+### 4.1. For Single Owner sudt
+
+````
+// Issue new sudt Info
+Inputs:
+	<... one of the input cells must have owner lock script as lock>
+Outputs:
+	udt_Cell:
+		Data:
+			amount: uint128
+		Type:
+			code_hash: simple_udt typescript
+			args: owner lock script hash (...)
+		Lock:
+			sudt creator defined
+Witnesses:
+	...
+	<Corresponding to owner cell>lock:signature,input_type:...,output_type:udt_info_data
+	...
+````
+
+For a single-owner-controlled sudt, info_data satisfies the following rules:
+
+- **Rule 1**: First, check according to the rules of info_cell, if there is a corresponding info_cell, use info_cell.
+- **Rule 2**: When there is no corresponding info_cell, scan the entire chain to find the first sudt issuance transaction that has data in the sudt info format in the output_type of the witness corresponding to owner_lock, and use the sudt info.
+
+### 4.2. For Script-drive sudt
+
+Script-driven sudt stands for, the owner_lock of sudt is not a lock controlled by a private key, but a special script.
+
+Since it is impossible to distinguish whether an owner lockscript is script-driven from the outside, it should be checked first according to the script-driven logic.
+
+````
+// Issue new Sudt/Sudt_Info
+Inputs:
+	<... one of the input cells must have owner lock script as lock>
+Outputs:
+	udt_Cell:
+		Data:
+			amount: uint128
+		Type:
+			code_hash: simple_udt typescript
+			args: owner lock script hash (...)
+		Lock:
+			<user defined>
+Witnesses:
+	...
+	<Corresponding to owner cell>lock:signature,input_type:...,output_type:udt_info_data
+	...
+````
+
+For a script-drive sudt, info_data satisfies the following rules:
+
+- **Rule 1**: First, check according to the rules of info_cell, if there is a corresponding info_cell, use info_cell.
+- **Rule 2**:  When there is no corresponding info_cell, scan the entire chain to find the first issuance transaction corresponding to sudt with data in the sudt info format in the output_type of the witness corresponding to the owner lockscript, and the hash of info_data is equal to the first 32 bytes of the owner lockscript args.
```

Comment:
The format of  "Rule X" is not uniform, sometimes is "Rule X:", sometimes is "RuleX", and here is "Rule X".